scenarioo-client/app/shared/navigation/branchOrderBy.filter.js

```diff
@@ -40,6 +43,40 @@ function scBranchOrderByFilter() {
             return 1;
         }
 
+        switch(order) {
+            case 'name-descending':
+                return orderByNameDescending(branchA, branchB);
+            case 'last-build-date-descending':
+                return orderByLastBuildDateDescending(branchA, branchB);
+            case 'name-ascending':  //also the default behavior
+            default:
+                return orderByNameAscending(branchA, branchB);
+        }
+    }
+
+    function orderByNameAscending(branchA, branchB){
+
+        // both are an alias or none is an alias -> use alphabetical ordering
+        var branchAName = branchA.branch.name.toLowerCase();
+        var branchBName = branchB.branch.name.toLowerCase();
+
+        if (branchAName < branchBName) {
+            return -1;
+        }
+
+        if (branchBName < branchAName) {
+            return 1;
+        }
+
+        return 0;
+    }
+
+    function orderByNameDescending(branchA, branchB){
+        return orderByNameAscending(branchA, branchB) * -1;
+    }
+
+    function orderByLastBuildDateDescending(branchA, branchB){
+
         if (branchA.alias === true) {
```

Comment:
this rule should be moved out, to have alphabetic sort for all aliases, not just for this sorting rule, see other comment